commons-civil/src/main/java/uk/gov/hmcts/reform/civil/documentmanagement/UnsecuredDocumentManagementService.java

```diff
@@ -91,6 +93,55 @@ public CaseDocument uploadDocument(String authorisation, PDF pdf) {
         }
     }
 
+    @Retryable(value = {DocumentUploadException.class}, backoff = @Backoff(delay = 200))
+    @Override
+    public CaseDocument uploadDocument(String authorisation, UploadedDocument uploadedDocument) {
+
+        String originalFileName = uploadedDocument.getFileBaseName();
+        log.info("Uploading file {}", originalFileName);
+
+        try {
+            MultipartFile file
+                = new InMemoryMultipartFile(FILES_NAME, originalFileName, MULTIPART_FORM_DATA_VALUE,
+                                            uploadedDocument.getBytes()
+            );
+
+            UserInfo userInfo = userService.getUserInfo(authorisation);
+            UploadResponse response = documentUploadClientApi.upload(
+                authorisation,
+                authTokenGenerator.generate(),
+                userInfo.getUid(),
+                documentManagementConfiguration.getUserRoles(),
+                Classification.RESTRICTED,
+                singletonList(file)
+            );
+
+            Document document = response.getEmbedded().getDocuments().stream()
```

Comment:
may we don't need this part.
If we get some problem in this documentUploadClientApi.upload method will throw some error. I believe so.
please could you check what happened when we have error on that point

Reply:
The error generally occurs at documentUploadClientApi.upload() - I've not seen it fail at this later point.
When it fails at documentUploadClientApi.upload(), I generally get this exception message back:
_org.camunda.bpm.extension.rest.exception.RemoteProcessEngineException: REST-CLIENT-001 Error during remote Camunda engine invocation of CaseDocumentClientApi#uploadDocuments(String,String,DocumentUploadRequest):_